app/src/main/java/com/github/se/orator/model/profile/UserProfile.kt

```diff
@@ -14,8 +14,20 @@ data class UserProfile(
     val bio: String? = null
 )
 
+enum class SessionType {
+  SPEECH,
+  INTERVIEW,
+  NEGOTIATION
+}
```

Comment:
To avoid using non-scalable `when` statements, I would recommend adding string attributes to each enum, and/or writing some functions in this enum class to retrieve the wanted strings associated to different values of different enums.

Reply:
This has been updated in the new push.